### DISCUSSION TITLE
Associations custom buttons

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#5412] [Help] [answered]
Given the example here: https://demo.filamentphp.com/shop/orders/1/edit?activeRelationManager=0

I would like to disable the edit and delete button. Instead I would like to add a custom text (something like "ban") which should execute an action on click which I will write manually. Is this possible?
Thank you in advance!

![image](https://user-images.githubusercontent.com/25671390/211167802-82a4ca8e-6c1a-42c1-9dfe-a75a85675ec1.png)

The same question applies to the repeater below. Can I even use it with associations as I want to automatically create a repeater from a relation.
![image](https://user-images.githubusercontent.com/25671390/211168177-9ed98e5d-4ff5-48b2-ba0d-59f5b25de44b.png)

Ok, right now I think I need to explain my problem better. Maybe this helps more to get to the core of the problem:

A game has many players. Each player is associated to a user and to a country (in the game). I want it to be possible to change the user who plays a country but I always want to show all countries.
```
| Country | User (Dropdown) |
| Country | User (Dropdown) |
| Country | User (Dropdown) |
```

As I wrote this down I had the idea to use an association for it and handle it over the edit form. Gonna give it a try over the weekend and eventually answer this discussion. Sorry for my "confuseness".

**Comment by danharrin** [comment, marked as answer]
You can remove the edit and delete actions from a table and add your own custom one - https://filamentphp.com/docs/2.x/tables/actions

Not possible to add actions to repeaters at the moment though
